qos_controller/serenity_controller.cpp

```diff
@@ -23,18 +22,25 @@ class SerenityControllerProcess :
     public Process<SerenityControllerProcess> {
  public:
   SerenityControllerProcess(
-      const lambda::function<Future<ResourceUsage>()>& _usage)
+      const lambda::function<Future<ResourceUsage>()> _usage)
       : usage(_usage) {}
 
   Future<list<QoSCorrection>> corrections() {
```

Comment:
I don't know which style guide we're using here. If we're using the Mesos one then all these braces should be on the next line?